test/e2e/dashboard_test.go

```diff
@@ -0,0 +1,64 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	monitoringdashboard "github.com/openshift/cluster-ingress-operator/pkg/operator/controller/monitoring-dashboard"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"testing"
+	"time"
+)
+
+func TestDashboardCreation(t *testing.T) {
```

Comment:
This test will fail in CI if the following holds true:

```go
func desiredMonitoringDashboard(ctx context.Context, infraStatus configv1.InfrastructureStatus) *corev1.ConfigMap {
	// If control plane topology is set to external, we do not deploy the dashboard
	if infraStatus.ControlPlaneTopology == configv1.ExternalTopologyMode {
		return nil
	}
```

We need to do the same check and not run the remainder of this test if there is no dashboard deployed.